InfiniTAM/CMakeLists.txt

```diff
@@ -33,10 +33,6 @@ find_package(libuvc QUIET)
 CANONIFY_BOOL(libuvc_FOUND)
 message(STATUS "libuvc found: ${libuvc_FOUND}")
 
-find_package(librealsense QUIET)
-CANONIFY_BOOL(librealsense_FOUND)
```

Comment:
These lower two lines are still required for the rest of infinitam to compile with librealsense, I only talked about the first line (find_package) that you can omit.